### PR TITLE
DOCSP-18692 landing page improvements

### PR DESCRIPTION
**Pull request by kyuan-mongodb**
## Pull Request Info

Improved the Node.js landing page based off the improvements made for the [Java page](https://docs.mongodb.com/drivers/java/sync/current/).

### Issue JIRA link:
https://jira.mongodb.org/browse/DOCSP-18692

### Snooty build log:
https://workerpool-boxgs.mongodbstitch.com/pages/job.html?jobId=6148a99ac608bec57f70c70b

### Docs staging link (requires sign-in on MongoDB Corp SSO):
https://docs-mongodbcom-staging.corp.mongodb.com/node/docsworker-xlarge/DOCSP-18692-LandingPageImprovements/

### Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [x] Did you run a grammar-check?
- [x] Does it render on staging correctly?
- [x] Are all the links working?
- [x] Are the staging and workerpool job links in the PR description updated?

### If your page documents a concept, does it meet the following criteria?

- [ ] Target the [Jasmin persona](https://drive.google.com/file/d/14FbBOLCVxwSP6M9BK4Nz1Ir9tzxT8_02/view)
- [ ] Target the [Lucas persona](https://drive.google.com/file/d/1J2vqJxo7ldv7OP_obA9Q-avf0o_ju4Lk/view)

### CODE REVIEW

File: source/index.txt [hunk 1]
```diff
@@ -23,91 +23,105 @@ MongoDB Node Driver
...
-technical information about classes, methods, and configuration objects
-within the MongoDB Node.js driver.
+
+The `MongoDB Node.js driver API documentation <{+api+}>`__ contains
+detailed information about classes and methods in the MongoDB
+Node.js driver.
 
 FAQ
 ---
+
```
Comment (melissamahoney-mongodb) [inline source/index.txt:68]: I'd suggest rewording similarly for FAQ, Issues & Help, Compatibility, and What's New. Perhaps you could give it a try before I make suggestions?

File: source/index.txt [hunk 2]
```diff
@@ -23,91 +23,105 @@ MongoDB Node Driver
...
 ----------
-The :doc:`What's New </whats-new>` section shows new features and
+
+The :doc:`What's New </whats-new>` section  lists new features and
 changes in each version.
+
+Learn
+-----
+
+Visit the Developer Hub and MongoDB University to learn more about the
```
Comment (melissamahoney-mongodb) [inline source/index.txt:95]: I like this, its action-oriented. Nice!

**Review by melissamahoney-mongodb — COMMENTED** [review]
Some suggestions to keep things action-oriented, and to avoid personifying the software.

File: source/index.txt [hunk 3]
```diff
@@ -23,91 +23,105 @@ MongoDB Node Driver
...
+methods.
 
 Fundamentals
 ------------
-Whether you are new to MongoDB or you just want to brush up on the core
-concepts, you can check out the **Fundamentals** section which features
-material on using the Node.js driver for the following:
+
+The Fundamentals section explains how to perform the following tasks
+using the Node.js driver:
```
[the remark this answers is not in the document]
Comment (kyuan-mongodb) [reply source/index.txt:55]: I did consider this and choose not to link it since it's only a drawer page while all the other pages aren't.

**Review by melissamahoney-mongodb — APPROVED** [review]
A really nice edit, Kailie! Good work!

**Review by terakilobyte — APPROVED** [review]
